policyengine_us/parameters/gov/fcc/lifeline/categorical_eligibility.yaml

```diff
@@ -4,16 +4,17 @@ values:
     - medicaid
     - snap
     - ssi
+    - is_head_start_income_eligible
```

Comment:
these three actually only apply to people on tribal lands. needs separate logic.

we also need to base head start categorical eligibility on participation (`head_start`) not only the income eligibility

Reply:
> these three actually only apply to people on tribal lands. needs separate logic.
> 
> we also need to base head start categorical eligibility on participation (`head_start`) not only the income eligibility

regarding your second point, the legal code states "only those [households] meeting its income qualifying standard", wouldn't that mean specifically head start income eligible

Reply:
yes, but it also says they must participate. someone income-eligible but who doesn't participate wouldn't qualify

Reply:
this PR only covers head start eligibility as we weren't able to find any explicit quantitative benefits of the program, for the categorical eligibility files I changed it to is_head_start_eligible but should I leave them out until we develop a head_start variable?

Reply:
Please either leave it out or point it to a head_start variable that doesn't yet have a formula. Pointing it to eligibility indicates a different policy.